mars_api/data_import/data_importer.py

```diff
@@ -0,0 +1,143 @@
+"""Like Transporter but without Jason Statham."""
+from sys import stdout
+
+from dateutil import parser
+from django.db.utils import IntegrityError
+
+from mars_api.serializers import GameSerializerForImportedData, PlayerScoreSerializer
+
+FIELD_REMAPPING = {
+    "game": {"map": "Default"},
+    "player_score": {
+        "corporation": {
+            "Interpl. Cinematics": "Interplanetary Cinematics",
+            "UNMI": "United Nations Mars Initiative",
+        }
+    },
+}
+
+
+def str_to_bool(arg):
+    return bool(int(arg))
+
+
+GAME_FIELD_MAPPING = {
+    0: {"id": int},
+    1: {"date": parser.parse},  # datetime
+    2: {"game_map": str},
+    3: {"prelude": str_to_bool},
+    4: {"colonies": str_to_bool},
+    5: {"number_of_generations": int},
+}
+
+PLAYER_SCORE_FIELD_MAPPING = {
+    7: {"player_nickname": str},
+    8: {"corporation": str},
+    10: {"terraform_rating": int},
+    11: {"milestones": int},
+    12: {"awards": int},
+    13: {"greeneries": int},
+    14: {"cities": int},
+    15: {"event_cards": int},
+    16: {"automated_cards": int},
+    17: {"active_cards": int},
+    18: {"resources": int},
+}
+
+
+def import_data(csv_file_path):
+    import csv
+
+    games = []
+    player_scores = []
+    current_game_id = None
+    with open(csv_file_path, newline="") as csvfile:
+        row_reader = csv.reader(csvfile, delimiter=",")
+        print_status("Parsing the csv file")
+
+        for row in row_reader:
+            if current_game_id != row[0]:
+                current_game_id = row[0]
+                games.append(create_game_dict(row))
+            player_scores.append(create_player_score_dict(row, current_game_id))
+
+    print_status(f"Number of games: {len(games)}; player scores: {len(player_scores)}")
+    save_data(games, player_scores)
+    stdout.write(format_message("Import Completed ✅", "\u001b[32;1m"))
```

Comment:
There are packages that do this - this would make the code rather unportable to other systems